### PR TITLE
Add swipe gesture to all eateries

### PR DESCRIPTION
**Pull request by jjennifergu**
## Overview

Finished changes from Antoinette's old branch on adding swipe gesture to all eateries. Created new branch because codebase organization was refactored.

## Changes Made

In addition to Antoinette's changes:
- Fix indexing of all eateries
- Disable swiping past the first/last page
- Add switch logic back to `didSelectRowAt` to only push eatery vc for eatery cards

## Test Coverage

Playtesting

## Related PRs or Issues 

Antoinette's PR: https://github.com/cuappdev/eatery-blue-ios/pull/16

## Screenshots (delete if not applicable)

https://user-images.githubusercontent.com/57200368/234445718-802553db-af83-43ab-b4b9-eba473ef27fe.mp4

### CODE REVIEW

File: Eatery Blue/UI/EateryScreen/EateryPageViewController.swift [hunk 1]
```diff
@@ -60,14 +60,24 @@ class EateryPageViewController: UIPageViewController {
 extension EateryPageViewController: UIPageViewControllerDataSource {
     func pageViewController(_ pageViewController: UIPageViewController, viewControllerBefore viewController: UIViewController) -> UIViewController? {
         guard let index = pages.firstIndex(of: viewController) else { return nil }
-        let previousIndex = abs((index - 1) % pages.count)
-        return pages[previousIndex]
+        let previousIndex = index - 1
+        if previousIndex >= 0 {
+            return pages[previousIndex]
+        }
+        else {
```
Comment (ReadeP) [inline Eatery Blue/UI/EateryScreen/EateryPageViewController.swift:67]: you don't need the else block here I think. Just return nil

File: Eatery Blue/UI/EateryScreen/EateryPageViewController.swift [hunk 2]
```diff
@@ -60,14 +60,24 @@ class EateryPageViewController: UIPageViewController {
...
     
     func pageViewController(_ pageViewController: UIPageViewController, viewControllerAfter viewController: UIViewController) -> UIViewController? {
         guard let index = pages.firstIndex(of: viewController) else { return nil }
-        let nextIndex = abs((index + 1) % pages.count)
-        return pages[nextIndex]
+        let nextIndex = index + 1
+        if nextIndex < pages.count {
+            return pages[nextIndex]
+        }
+        else {
```
Comment (ReadeP) [inline Eatery Blue/UI/EateryScreen/EateryPageViewController.swift:78]: same here

**Review by ReadeP — APPROVED** [review]
Looks good! :D

**Review by antoinztte — APPROVED** [review]
LGTM!

**Pull request merged**